src/sssom/context.py

```diff
@@ -16,7 +16,7 @@
 # EXTERNAL_CONTEXT_PATH = HERE / "obo.epm.json"
 
 SSSOM_URI_PREFIX = "https://w3id.org/sssom/"
-SSSOM_BUILT_IN_PREFIXES = ("sssom", "owl", "rdf", "rdfs", "skos", "semapv")
+SSSOM_BUILT_IN_PREFIXES = ("sssom", "owl", "rdf", "rdfs", "skos", "semapv", "orcid")
```

Comment:
Is `orcid` now supposed to be a built-in prefix in SSSOM?

If so, can you please make sure that [the specification](https://mapping-commons.github.io/sssom/spec/#tsv) is updated accordingly? Currently, the spec still stays that only `sssom`, `owl`, `rdf`, `rdfs`, `skos`, `semapv` are built-in.

Reply:
Hmm, it hadn't occurred to me that this was part of the spec and not an implementation detail. I don't have the motivation to get into a discussion about this so I will change the PR

Reply:
The list of built-in prefixes is part of the spec because it wouldn’t be great if a given prefix was considered built-in by one implementation and not built-in by another. A mapping set that makes use of that prefix without explicitly declaring it (thinking it is universally built-in) would be accepted by the first implementation and rejected by the second.